packages/cli/src/lib/transform.ts

```
@@ -118,31 +124,31 @@ export function transform(
   }
-  
+
   log(`Found ${targetFiles.length} relevant files for codemod '${codemod}'`);
-  
+
   const command = buildCommand(codemodPath, targetFiles, transformOptions);
-  
+
   // Use spawn instead of execFileSync to capture output in real-time
   if (options.onProgress) {
     const result = spawnSync(command[0]!, command.slice(1), {
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)